tests/code_wars/test_check_same_case.py

```diff
@@ -12,19 +12,18 @@
 # 'B' and 'g' returns 0
 # '0' and '?' returns -1
 
-from code_challenges.code_wars.check_same_case import case_check
+from code_challenges.code_wars.check_same_case import evaluate_characters_case
 
 def test_case():
```

Comment:
The only thing I was suggesting needed to be changed was `test_case` to `test_check_same_case`

Now it would be `test_evaluate_characters_case`